multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/cf/CloudFoundryClientFactory.java

```diff
@@ -46,35 +49,44 @@ private ImmutableCloudControllerRestClientFactory createClientFactory(Applicatio
     @Override
     protected CloudControllerClient createClient(CloudCredentials credentials) {
         OAuthClient oAuthClient = oAuthClientFactory.createOAuthClient();
-        List<ExchangeFilterFunction> exchangeFilters = getExchangeFiltersList(null, null);
+        List<ExchangeFilterFunction> exchangeFilters = getExchangeFiltersList(null, null,null);
```

Comment:
Now I am thinking that this was not good idea to add just another null. Maybe we have to create separate method without parameters.